### ISSUE TITLE
Add more info to readme

### ISSUE THREAD

**Issue by Lucretia** [Ironclad-Project/Gloire#11]
You should point out that the user needs to be part of the kvm group:

```bash
sudo usermod -aG kvm <user>
```

and that raw should be specified for the disk:

```bash
qemu-system-x86_64 -enable-kvm -m 2G -M q35 -drive format=raw,file=gloire-20240615.img -serial stdio
```

**Comment by streaksu** [comment]
I agree with and will add the `usermod` part. What change does the `raw` format actually do?

**Comment by mintsuki** [comment]
The `raw` thing just gets rid of a harmless warning which has existed in qemu for years but does nothing. It says it prevents writes to the first sector, but it really does nothing, it's completely unwired.

**Comment by mintsuki** [comment]
Also the `kvm` group thing really depends on the distro, it's not the same everywhere.

**Comment by streaksu** [comment]
> The `raw` thing just gets rid of a harmless warning which has existed in qemu for years but does nothing. It says it prevents writes to the first sector, but it really does nothing, it's completely unwired.

I see, I guess there is doing it anyways to get rid of the warning, thanks for the clarification

> Also the kvm group thing really depends on the distro, it's not the same everywhere.

Yeah I'll make sure to mention it

**Comment by streaksu** [comment]
Fixed in ece22c40b42603eb914147d6b1f77dfed4a63c7a.

**Comment by Lucretia** [comment]
> I agree with and will add the `usermod` part. What change does the `raw` format actually do?

qemu generates an error saying that the fs will be read only and it won't boot.
